apis/controller/v1alpha1/devworkspacerouting_types.go

```diff
@@ -25,8 +25,6 @@ type DevWorkspaceRoutingSpec struct {
 	DevWorkspaceId string `json:"devworkspaceId"`
 	// Class of the routing: this drives which DevWorkspaceRouting controller will manage this routing
 	RoutingClass DevWorkspaceRoutingClass `json:"routingClass,omitempty"`
-	// Routing suffix for cluster
-	RoutingSuffix string `json:"routingSuffix"`
```

Comment:
Worth noting that this is a breaking change for v1alpha1 devworkspaceroutings, but I don't think it impacts anything as nothing is deploying devworkspaceroutings yet.